environment.yml

```diff
@@ -2,17 +2,18 @@
 name: streamlit-env
 channels:
    - conda-forge
+   - bioconda
 dependencies:
    - python==3.11
    - plotly==5.22.0
    - pip==24.0
    - numpy==1.26.4 # pandas and numpy are dependencies of pyopenms, however, pyopenms needs numpy<=1.26.4
-   - mono==6.12.0.90
+
    - pip:
         # dependencies only available through pip
         # streamlit dependencies
         - streamlit>=1.38.0
         - captcha==0.5.0
+        - pyopenms==3.2.0
```

Comment:
pyopenms does not need to be added here. Locally the installation recommends using `requirements.txt. The `environment.yml` is used by docker where we build `pyopenms` within the image.